Add Python 3.12 support (#154)
iqm-finland/iqm-client: CHANGELOG.rst

Changelog
=========

Version 20.9
============

> * Add Python 3.12 support

Reviewer: ```suggestion
* Added Python 3.12 support `#154 <https://github.com/iqm-finland/iqm-client/pull/154>`_
```

> * Support for Python 3.9 will be dropped in a later major release.

Reviewer: ```suggestion
* Python 3.9 support is deprecated and will be removed in the future
```
